### PR TITLE
Initialize with basic folders and README explaining what each is.

### PR DESCRIPTION
**Pull request by lylemoffitt**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Docs Folder
+
+This directory is for documentation. We'll figure that out later. It'll probably be a combination of one or more of the following.
+
+## C++ Source Docs
+
+This is will be the primary concern. We will use one of the following tools:
```
Comment (russellkir) [inline docs/README.md]: ~~is~~

File: src/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+This directory is for source files
+
+## Format
+
+There are lots of ways to organize source files. For now, let's keep things simple. We can refactor later.
+
+- Regular C++ files:
+    - `src/*.cpp`
+    - `src/*.h`
```
Comment (russellkir) [inline src/README.md:11]: Do we want this in an include directory instead?

**Review by russellkir — APPROVED** [review]

File: src/README.md (hunk 2 above)
Comment (lylemoffitt) [reply src/README.md:11]: IMO it's only necessary when you have headers you want to expose to other projects so they can link against yours. Like, if you're publishing a `*.so`. We can always refactor later. I like to do that around the time the code base develops a more complex hierarchy.

Comment (russellkir) [reply src/README.md:11]: Sounds good, was just asking

**Pull request merged**